docs/en/05_Releases_and_changelogs/cwp_2.3.0.md

```diff
@@ -34,10 +34,13 @@ can be made:
 "prefer-stable": true
 ```
 
-### Bootstrap 4
+### Major Theme Updates
```

Comment:
Super minor suggestion, but other headings only capitalise the first word e.g. Known issues below